package/tests/test_api/test_rest/test_reponses.py

```diff
@@ -8,6 +8,19 @@
 from kedro_viz.api import apps
 from kedro_viz.models.flowchart import TaskNode
 
+try:
+    from kedro_datasets import pandas  # isort:skip
+except ImportError:
+    from kedro.extras.datasets import (  # Safe since ImportErrors are suppressed within kedro.
+        pandas,
+    )
+
+
+def _get_dataset_type(dataset):
```

Comment:
This function has been declared twice with the same body (here and in `test_queries`), I am fine with the DAMP approach, but ditto my comment.

Reply:
so we don't have a utils file where we can have some common test functions -- one function is for flowchart and the other is for experiment tracking hence there is duplicity.